tests/python_functional/src/syslog_ng_config/statements/sources/source_driver.py

```diff
@@ -20,15 +20,26 @@
 # COPYING for details.
 #
 #############################################################################
+import logging
 
 
 class SourceDriver(object):
     group_type = "source"
 
-    def __init__(self, positional_parameters=None, options=None):
+    def __init__(self, logger_name=__name__, positional_parameters=None, options=None):
```

Comment:
Related to logger_name: same as above. Please let's remove it, or think it over (I will help).

Reply:
Removed.